tiflash: Add disaggregated arch related configs (#22777)
pingcap/docs: tiflash/tiflash-configuration.md

- Specifies the minimum ratio of valid data in a PageStorage data file. When the ratio of valid data in a PageStorage data file is less than the value of this configuration, GC is triggered to compact data in the file.
- Default value: `0.5`

##### `remote_checkpoint_interval_seconds` <span class="version-mark">New in v9.0.0</span>

> - Controls the execution interval of the background task for TiFlash to upload checkpoints to object storage. This parameter only takes effect on TiFlash Write Nodes that have object storage enabled in the [disaggregated storage and compute architecture](/tiflash/tiflash-disaggregated-and-s3.md).

Reviewer: ```suggestion
- Controls the execution interval of the background task for TiFlash to upload checkpoints to object storage. This parameter only takes effect on TiFlash Write Nodes that store data on object storage in the [disaggregated storage and compute architecture](/tiflash/tiflash-disaggregated-and-s3.md).
```

- Default value: `30`
- Unit: seconds
- Decreasing this value allows the latest data to be uploaded to object storage faster, but increases the frequency of checkpoint uploads and associated overhead. Increasing this value reduces the upload frequency. It is generally not recommended to modify this parameter.

##### `remote_gc_method` <span class="version-mark">New in v9.0.0</span>

- Controls how TiFlash executes GC on object storage. This parameter only takes effect on TiFlash Write Nodes in the [disaggregated storage and compute architecture](/tiflash/tiflash-disaggregated-and-s3.md).

> - Optional values:

Author: ```suggestion
- Value options:
```

- Default value: `1`
- This parameter only affects how remote objects are recycled and does not affect TiFlash's behavior of uploading checkpoints. If you are not sure whether TiFlash is compatible with the lifecycle rules of your object storage, it is recommended to set this value to `2` during deployment to use the `ScanThenDelete` method for object storage GC.

##### `remote_gc_interval_seconds` <span class="version-mark">New in v9.0.0</span>

- Controls the execution interval of background tasks for remote object storage GC. This parameter only takes effect on TiFlash Write Nodes in the [disaggregated storage and compute architecture](/tiflash/tiflash-disaggregated-and-s3.md), and only the node currently selected as the GC owner actually executes this task.
- Default value: `3600`
- Unit: seconds
- When `remote_gc_method` is set to `1`, this parameter controls the frequency at which TiFlash scans locks and manifests and maintains lifecycle rules, but the actual physical deletion time of objects is still determined by the lifecycle rules of the object storage.

> - When `remote_gc_method` is set to `2`, this parameter also controls the frequency at which TiFlash scans and directly deletes expired objects. It is generally not recommended to modify this parameter.

Author: ```suggestion
- When [`remote_gc_method`](#remote_gc_method-new-in-v900) is set to `1`, this parameter controls the frequency at which TiFlash scans locks and manifests and maintains lifecycle rules, but the actual physical deletion time of objects is still determined by the lifecycle rules of the object storage.
- When [`remote_gc_method`](#remote_gc_method-new-in-v900) is set to `2`, this parameter also controls the frequency at which TiFlash scans and directly deletes expired objects. It is generally not recommended to modify this parameter.
```
